### ISSUE TITLE
Slider.kz not answering requests

### ISSUE THREAD

**Issue by pythoncat1** [Shubhamrawat5/apple-playlist-downloader-webapp#3]
slider.kz changed the way they handled routing and also added cloudflare, disabling this tool

**Comment by Shubhamrawat5** [comment]
Yeah I know that, I'm thinking to use ytdl module to download song now, will implement soon

**Comment by pythoncat1** [comment]
Its unfortunate that slider.kz uses cloudflare now… Youtube’s audio quality is way lower than slider’s
